Pip support

nhumrich opened the pull request:
This pull request does the following things:
1. fix bug in ASCII_TO_NUMBER dict
2. changes formatting for Pep8 compatibility
3. Adds support for Python3 without breaking python 2.7+
4. Adds support for pip

You can upload to pip using `./setup.py sdist upload`. In order to upload, you will need to change the version `__version__` in gitfiti.py. You may want to take the liberty of editing some things in setup.py

I have taken the privileged of uploading this to pypi already. If you send me your pypi username I will pass ownership to you. 
The readme will have to be updated to show you can install with pip

gelstudios commented:
Would you mind splitting this into two PR's? one for the pip __setup__ addition and the other rebased against master? I merged a previous PR that already cleaned up PEP8 violations and py2/3 support.

Thanks!!!

My pypi username is also "gelstudios".

homeworkprod commented on gitfiti.py:
```diff
@@ -98,7 +98,7 @@
 
 ASCII_TO_NUMBER = {
-  '_': 0,
+  '-': 0,
```
Ha, well spotted!

That should explain why I had to replace 1s with 0s in e7e67834c873a68fd8ec2dcfbdcd1429660126e3!

Please rebase and adjust the `NUMBERS` two-dimensional list in the test so that it contains 1s instead of 0s again like the original `ONEUP` value (that should make the test pass again).

homeworkprod commented:
Thanks for creating the setup script!

Feel free to provide that in a separate PR.

nhumrich committed:
Support for pip

Had to move gitfiti.py into a module for the entrypoint to work correctly
Also added setup.py for pip support

nhumrich committed:
fix small bug with the ASCII_TO_NUMBER dict

nhumrich committed:
more pep8 changes

nhumrich commented:
updated. This is now just mostly pip support

nhumrich commented:
@gelstudios you are now an owner of the package. Feel free to remove me if youd like.

homeworkprod commented:
You might want to add a Trove classifier for Python 3.4 as well.

Thanks for introducing a package, was going to do that later, too. This makes it easier to deconstruct code into several modules.

I assume the test for `str_to_sprite` will break until adjusted.

nhumrich committed:
Merge branch 'master' of https://github.com/gelstudios/gitfiti

nhumrich committed:
fix test

nhumrich commented:
ok fixed test and added the classifier
